cmd/cluster/create_cluster.go

```diff
@@ -158,7 +158,7 @@ func init() {
 	createClusterCmd.MarkFlagRequired("cluster-name")
 	createClusterCmd.Flags().StringToString("credentials", nil, `[REQUIRED] Credentials to login to the cluster. Please provide key value pairs username=<user-name>,password=<password>.`)
 	createClusterCmd.MarkFlagRequired("credentials")
-	createClusterCmd.Flags().String("cloud-type", "", "[OPTIONAL] The cloud provider where database needs to be deployed. AWS or GCP. Default AWS.")
+	createClusterCmd.Flags().String("cloud-provider", "", "[OPTIONAL] The cloud provider where database needs to be deployed. AWS or GCP. Default AWS.")
```

Comment:
@shinji62 Do you think we should allow both `cloud-type` and `cloud-provider` to ensure backward compatibility for cluster worfklows too similar to what you did for read replicas?

Reply:
I think this is OK to break right now.